sdks/wasm/src/linker-subs.xml

```diff
@@ -88,11 +88,16 @@
 	  <!-- Keeps CultureInfo::get_LCID alive -->
 	  <method signature="System.Void OnSerializing(System.Runtime.Serialization.StreamingContext)" body="remove"/>
 	</type>
+	<type fullname="System.SR">
+	  <!-- Keeps resource manager alive -->
+	  <method signature="System.Boolean UsingResourceKeys()" body="stub" value="true"/>
+	</type>
   </assembly>
 
   <assembly fullname="System.Console">
 	<type fullname="System.SR">
 	  <!-- Keeps resource manager alive -->
+	  <!-- FIXME: The resources are still kept -->
```

Comment:
The meaning of this seems like it might be unclear without additional context (like from the issues about this I've previously read) so it might be nice to be slightly more detailed here, since someone less familiar might think "resx? data file? blobs section in the assembly?" when they see "resources"

Reply:
This is the resources used by System.SR.